README.md

```diff
@@ -90,3 +90,44 @@ await mongoose.connect(url, defaultMongoOpts)
 const person = await PersonRepo.byID(id));
 const people = await PersonRepo.all({});
 ```
+
+## How to run a migration
+
+```shell
+yarn migrations init
+```
+
+The above command will initialize a `migrate-mongo-config.js` in the root directory and create a migration folder in your `src` folder.
+
+Now run
+
+```shell
+yarn migrations create [description]
```

Comment:
You could be a bit more descriptive about what a description is. You could just put an example name there.